os/board/rtl8730e/src/rtl8730e_alc1019.c

```diff
@@ -62,6 +62,10 @@ extern FAR struct i2s_dev_s *amebasmart_i2s_initialize(uint16_t port, bool is_re
 /*other pin config */
 #define ALC1019_GPIO_RESET_PIN		PA_22
 
+#if defined(CONFIG_AUDIO_NDP120)
```

Comment:
Actually alc1019 doesn't control this pin by itself, but this is required to support AEC, between amp & ndp120.
Anyway As I remember, this gpio pin will not be changed, then I think it is need to be moved to rtl8730e_ndp120.c

Reply:
Added a commit that moves the enabling of DMIC via PA_25 to `rtl8730e_ndp120_initialize()` function instead